examples/examples.py

```diff
@@ -199,11 +199,18 @@ def stft_plot(signal, sample_rate=DEFAULT_SAMPLE_RATE):
 #    print(f"{p} grad1={adsr.torchparameters[p].data.grad} grad2={adsr.torchparameters[p].data.grad}")
 # -
 
+# **Generating Random Envelopes**
+#
+# If we don't set parameters for an ADSR, then the parameters will be random when
+# initialized. We can also pass in an upsample attribute to allow control signal
+# modules to upsample to full sample rate.
+
 # Note that module parameters are optional. If they are not provided,
 # they will be randomly initialized (like a typical neural network module)
-adsr = ADSR(synthglobals, device)
+adsr = ADSR(synthglobals, device=device, upsample=True)
```

Comment:
Just a quick question, why would we want upsample=False?

Reply:
I think you saw below -- but when we modulate another control rate module like LFO we need the output to be control rate.

Reply:
Got it yeah